src/utils/validators.js

```diff
@@ -121,6 +125,23 @@ export const getWalletNameError = (
   }
 }
 
+export const getUnstoppableDomainAddress = async (domain: string) => {
+  try {
+    return await new Resolution().addr(domain, 'ADA')
+  } catch(e) {
+    switch(e.code) {
+      case 'UnsupportedDomain':
+        throw new Error('{"unsupportedDomain": true}');
+      case 'RecordNotFound':
+        throw new Error('{"recordNotFound": true}');
+      case 'UnregisteredDomain':
+        throw new Error('{"unregisteredDomain": true}');
+      default:
+        throw new Error('{"invalidAddress": true}');
+    }
+  } // invalid domain
+}
+
```

Comment:
Cant I get an opinion on this change?
Flow doesn't like objects in `e.message`.
I believe this change, thus necessitating `JSON.parse()` in `SendScreen.recomputeAll()` is slower than just passing the object literal.

What are your opinions on this?